### PR TITLE
Refine around naming in metadata.

### PR DESCRIPTION
**Pull request by ueshin**

### CODE REVIEW

File: databricks/koala/structures.py [hunk 1]
```diff
@@ -195,6 +195,10 @@ def getField(self, name):
                     "Field {} not found, possible values are {}".format(name, ", ".join(fnames)))
             return anchor_wrap(self, self._spark_getField(name))
 
+    # TODO: automate the process here
+    def alias(self, name):
```
Comment (thunterdb) [inline databricks/koala/structures.py]: With this fix, I am not even sure we need to overwrite this function.

Comment (ueshin) [reply databricks/koala/structures.py]: I just removed for now, but if we have this, `col.alias("x").name` will return `x`.
Currently the original `Column` methods are not anchor the dataframe reference except for methods in `_wrap_operators`.
We might need to wrap them automatically as the same.

**Comment by ueshin** [comment]
@thunterdb Do you mean the changes with the automatic wrapping or just with `alias()` for now?

**Comment by thunterdb** [comment]
@ueshin  I meant just making sure that chaining method calls with .alias works. This should fail without this PR and then work with this PR.

**Comment by thunterdb** [comment]
@ueshin thanks, merging.

**Comment by thunterdb** [comment]
@ueshin I reverted this PR, since it broke the build. Can you try again? There were some errors with the master merge.

**Comment by ueshin** [comment]
I see, I'll check and submit a PR again.

**Pull request merged**